ats/src/atsMachines/lsf_asq.py

```diff
@@ -570,12 +575,12 @@ def calculateCommandList(self, test):
                         str_lrun_jsrun_args = ""
 
                     if lsfMachine.debugJsrun:
-                        print "JSRUN 110 DEBUG lsf_asq test.jsrun_bind_r       =  %s " % test.jsrun_bind_r
-                        print "JSRUN 110 DEBUG lsf_asq test.jsrun_bind_none    =  %s " % test.jsrun_bind_none
-                        print "JSRUN 110 DEBUG lsf_asq test.jsrun_bind         =  %s " % test.jsrun_bind
-                        print "JSRUN 110 DEBUG lsf_asq self.mpibind            =  %s " % self.mpibind
-                        print "JSRUN 110 DEBUG lsf_asq str_lrun_jsrun_args     =  %s " % str_lrun_jsrun_args
-                        print "JSRUN 110 DEBUG lsf_asq cpu_per_rs              =  %d " % cpu_per_rs
+                        print("JSRUN 110 DEBUG lsf_asq test.jsrun_bind_r =  %s " % test.jsrun_bind_r)
```

Comment:
were these spaces removed on purpose?

Reply:
That was a side effect of using Python's "2to3" module.
Ideally we would replace the spaces with proper [Python 2.7 string formatting](https://docs.python.org/2.7/library/stdtypes.html#string-formatting-operations) and let the interpreter append the padded the spaces. Or better yet make use of [Python 3.6+ f strings](https://realpython.com/python-f-strings/).